test/test_helper.rb

```diff
@@ -0,0 +1,29 @@
+ENV['RAILS_ENV'] ||= 'test'
+$:.unshift File.dirname(__FILE__)
+
+require 'rails_app/config/environment'
+require 'rails/test_help'
+require 'database_cleaner'
+require 'minitest/rails/capybara'
+require 'mongoid'
+require 'mongoid-history'
+require 'mongoid_search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.